### PR TITLE
Issues/6600

### PR DESCRIPTION
**Pull request by jagraz**
Updates to the DefaultLocalizedStringManager.cs file to only harvest dictionaries from active themes.

### CODE REVIEW

File: src/Orchard/Localization/Services/DefaultLocalizedStringManager.cs [hunk 1]
```diff
@@ -28,13 +31,15 @@ public class DefaultLocalizedStringManager : ILocalizedStringManager {
             ICacheManager cacheManager,
             ILocalizationStreamParser locationStreamParser,
             ShellSettings shellSettings,
-            ISignals signals) {
+            ISignals signals,
+            ShellDescriptor shellDescriptor) {
```
Comment (sebastienros) [inline src/Orchard/Localization/Services/DefaultLocalizedStringManager.cs]: Are you sure the `ShellDescriptor` only contains the enabled features? There is also `IFeatureManager` which contains a `GetEnabledFeatures`. Also there is a mismatch between enabled features and localization files are these files don't mention features per se, just the module, but I am not sure you can do anything about it.

**Comment by sebastienros** [comment]
If that fixes the issue, then GTG.

**Comment by jagraz** [comment]
Yeah the GetEnabledFeatures won't show themes. Shell Descriptor only shows features and themes that are enabled.

**Comment by sebastienros** [comment]
This is really not intuitive ;)

**Comment by sebastienros** [comment]
Are you sure you don't want it in 1.10.x ?

**Comment by jagraz** [comment]
it's up to you guys. I can put it in 1.10.1. if you feel comfortable with it.

**Pull request closed without merging**